docs/index.html

```diff
@@ -179,7 +179,12 @@ <h3 id="supported-flags">Supported Flags</h3>
         
           <div class='highlight'><pre>command.inventory(<span class="hljs-string">'/etc/ansible/hosts'</span>)</pre></div>
         
-      
+
```

Comment:
No need to change the actual HTML file only examples/index.js. Docs are auto generated.